website/content/docs/internals/limits.mdx

```diff
@@ -285,3 +285,21 @@ The current number of unexpired leases can be monitored via the
 The Transform secret engine obeys the [FF3-1 minimum and maximum sizes
 on the length of an input](/docs/secrets/transform#input-limits), which
 are a function of the alphabet size.
+
+### Plugin limits
```

Comment:
If this entire section applies only to external plugins, then perhaps the section should be called "External plugin limits"? Currently the applicability is sort of buried.

Reply:
Updated the section title on cde03f0.